.vib/cassandra/goss/vars.yaml

```diff
@@ -0,0 +1,29 @@
+binaries:
+  - cassandra
+  - cqlsh
+  - python
+  - java
+  - gosu
+  - yq
+files:
+  - mode: "0755"
+    paths:
+      - /opt/bitnami/cassandra/bin/cqlsh
+directories:
```

Comment:
I would add a check for the `/opt/bitnami/cassandra/conf.default` directory created at compilation time.